presentation-api/controlling-ua/getAvailability_sandboxing_success.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Sandboxing: Retrieving display availability from a nested context succeeds when allow-presentation is set</title>
+<link rel="author" title="Francois Daoust" href="https://www.w3.org/People/#fd">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dom-presentationrequest-getavailability">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    async_test(function (t) {
+        function startWhenReady(ev) {
+            var childFrame = document.getElementById('childFrame');
+            if (ev.data === 'ready') {
+                window.removeEventListener('message', startWhenReady);
+                childFrame.contentWindow.postMessage('getAvailability', '*');
+                window.addEventListener('message', t.step_func(function (ev) {
+                    assert_equals(ev.data, 'success',
+                        'Presentation sandboxing did not work as expected.');
```

Comment:
Could you clarify what you mean by "check"?

If you mean "ensure the assertion fails", the test fails in all browsers, except Chrome and Canary. For instance, in Edge, which does not implement the Presentation API, I get:

> assert_equals: Presentation sandboxing did not work as expected. expected "success" but got "Could not create PresentationRequest"

This test is meant to check that a controlling user agent supports the `allow-presentation` flag and correctly allows nested contexts to use the Presentation API when this flag is set. The test passes in the current version of Chrome v52 and in Canary v55, although for different reasons:
* Chrome v52 does not support the `allow-presentation` sandbox flag but allows nested contexts to call the Presentation API (in effect, there is no sandboxing). This gets detected because Chrome fails the companion `_error` test.
* Canary v55 correctly supports the `allow-presentation` sandbox flag.

For the time being, there are no browsers that:
1. support the Presentation API
2. do not support the `allow-presentation` flag
3. forbid calls to the Presentation API from nested contexts

Now, if you rather mean that the message should be clearer, I may update it, e.g. to "Call to getAvailability did not succeed."

Or is it something else?